backend/app/elastic_search/connect.py

```diff
@@ -68,8 +68,9 @@ def search_index(es_client, index_name, query):
         (For more details, refer to https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html)
     """
     try:
-        res = es_client.search(index=index_name, query=query)
-        logger.info("Got %d Hits:" % res["hits"]["total"]["value"])
+        res = es_client.msearch(index=index_name, searches=query)
```

Comment:
this is multi-search instead of search. I didn't read to much about it but that's the endpoint that the frontend react library is calling under the hood.
https://www.elastic.co/guide/en/elasticsearch/reference/current/search-multi-search.html